packages/@angular/cli/models/command-runner.ts

```diff
@@ -131,23 +135,17 @@ function parseOptions(parser: Function,
     .filter(key => key.indexOf('-') !== -1)
     .forEach(key => delete parsedOptions[key]);
 
-  return parsedOptions;
-}
-
-// Map arguments to options.
-function mapArguments(options: any, args: string[]): any {
-  const optsWithMappedArgs = {...options};
-  optsWithMappedArgs._.forEach((value: string, index: number) => {
+  parsedOptions._.forEach((value: string, index: number) => {
     // Remove the starting "<" and trailing ">".
-    const arg = args[index];
+    const arg = commandArguments[index];
     if (arg) {
-      optsWithMappedArgs[arg] = value;
+      parsedOptions[arg] = value;
     }
   });
 
-  delete optsWithMappedArgs._;
+  delete parsedOptions._;
```

Comment:
I've been thinking about this one... we should only delete if all the values have been mapped appropriately? Although you've allowed for the rawArgs to live on, just something to consider.

Reply:
I'm going to fix the argv before RC, since we have one more week.